pkg/functions/riff/builder_test.go

```diff
@@ -0,0 +1,27 @@
+///////////////////////////////////////////////////////////////////////
+// Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+///////////////////////////////////////////////////////////////////////
+package riff
```

Comment:
Could we have a "common" test suite against the different drivers and builders?  Obviously they would have external dependencies, but it may be a good integration/e2e type test.

Reply:
I'll think about it. I try to test things bottom up. But we sure need e2e tests for these.